styles.css

```diff
@@ -1,70 +1,25 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;1,100&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600&family=Poppins:ital,wght@0,100;0,200;1,100&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
 
-* {
-  margin: 0;
-  padding: 0%;
-  box-sizing: border-box;
-  text-decoration: none;
-  list-style-type: none;
-}
-  
-body {
-  font-family: 'Roboto', sans-serif;
-  line-height: 20px;
-  background-color: #e5e5e5;
-}
+
 
 /* For Mobile */
 
-@media screen and (max-width: 768px) {      
-.header {
-  background-color: #3C3A39;
-  height: 92px;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  font-weight: bold;
-  padding-top: 20px;
-}
-      
-.logo a{
-  color: white;
-  margin-left: 20px;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 24px;
-}
```

Comment:
- [ ] Kindly follow [this link](https://www.freecodecamp.org/news/taking-the-right-approach-to-responsive-web-design/#:~:text=What%20is%20mobile%2Dfirst,styling%20for%20larger%20screen%20sizes.&text=In%20that%20example%2C%20for%20small,re%20simply%20applying%20some%20padding.) on mobile-first design. That is probably why your site is overflowing at a width of 375px. You should be using min-width not max-width.

Reply:
![375](https://user-images.githubusercontent.com/89055502/178050756-09282c1e-26ec-49b1-9420-6b0d61d67527.png)
this is my portfolio on screen size of 375px, it is perfectly fine, i am totally unaware of why it showed that way when you checked

Reply:
@fobadara